src/Core/Prefab/GlobeView.js

```diff
@@ -163,7 +163,7 @@ function GlobeView(viewerDiv, coordCarto, options) {
             this._latestUpdateStartingLevel = commonAncestor.level;
             return [commonAncestor];
         } else {
-            return [];
+            return layer.level0Nodes;
```

Comment:
I see. It seems that L158 saves me a little though (when sources does not have materials).

Actually, I think the best safeguard would be to test `source.layer === 'globe'`, because of line 137. WDYT?